examples/tv-app/tv-common/clusters/messages/MessagesManager.h

```diff
@@ -21,6 +21,97 @@
 
 #include <iostream>
 #include <list>
+#include <vector>
+
+/**
+ * Note on memory management:
+ *
+ * The CachedMessage contains strings and objects
+ * allocated when created (via HandlePresentMessagesRequest) and freed
+ * in the destructor (via HandleCancelMessagesRequest).
+ *
+ */
+
+struct CachedMessageOption
+{
+    uint32_t mId;
```

Comment:
Why do you need the separate mId member?

Reply:
removed